components/brave_rewards/resources/rewards_page/stories/storybook_strings.ts

```diff
@@ -0,0 +1,210 @@
+  notificationMonthlyTipCompletedText: 'Your monthly contributions have been processed.',
+  notificationMonthlyTipCompletedTitle: 'Contributions',
+  onboardingButtonLabel: 'Start using Brave Rewards',
+  onboardingErrorCountryDeclaredText: 'Unfortunately, there was an error while trying to set your country. Please try again.',
+  onboardingErrorDisabledText: 'New signups for Brave Rewards are currently disabled in your region. However, you can always try again later. $1Learn more$2',
+  onboardingErrorDisabledTitle: 'Brave Rewards not available',
+  onboardingErrorText: 'Unfortunately, there was an error while trying to set up Brave Rewards. Please try again.',
+  onboardingErrorTitle: 'Something went wrong',
+  onboardingLearnMoreLabel: 'How does it work?',
+  onboardingSuccessLearnMoreLabel: 'How does Brave Rewards work?',
+  onboardingSuccessText: 'Now you can get rewarded for viewing ads. You’re helping make the web a better place for everyone. And that’s awesome!',
+  onboardingSuccessTitle: 'Woohoo! You\'re all set!',
+  onboardingTermsText: 'By enabling Rewards, you agree to the $1Terms of Service$2 and $3Privacy Policy$4.',
+  onboardingTextItem1: 'Get paid for private ads you see in Brave',
+  onboardingTextItem2: 'Get special benefits and discounts',
+  onboardingTitle: 'Turn on Brave Rewards',
+  payoutAccountBalanceLabel: 'Your balance',
+  payoutAccountConnectedLabel: 'Connected',
+  payoutAccountDetailsTitle: '$1 wallet',
+  payoutAccountLabel: 'Payout account',
+  payoutAccountLink: 'Go to my account',
+  payoutAccountLoggedOutTitle: 'Logged Out',
+  payoutAccountLoginButtonLabel: 'Log in to $1',
+  payoutAccountLoginText: 'Log in to $1 to see your balance',
+  payoutAccountTitle: 'Payout Account',
+  payoutAccountTooltip: 'Your BAT earnings will be paid to this payout account.',
+  payoutCheckStatusLink: 'Check status',
+  payoutCompletedText: 'The payout for $1 rewards has completed.',
+  payoutPendingText: 'The payout for $1 rewards will begin processing by $2.',
+  payoutProcessingText: 'The payout for $1 rewards is in progress.',
+  payoutSupportLink: 'Support',
+  recurringListEmptyText: 'You don\'t have any recurring contributions set up yet.',
+  recurringNextContributionLabel: 'Next contribution:',
+  recurringTitle: 'Recurring Contributions',
+  removeButtonLabel: 'Remove',
+  resetButtonLabel: 'Reset',
+  resetConsentText: 'I understand that my current Brave Rewards profile and Brave Rewards data will be deleted from my device, and I\'ve read the $1support article about resetting$2.',
+  resetRewardsButtonLabel: 'Disable and reset Rewards',
+  resetRewardsText: 'By resetting, your current Brave Rewards profile will be deleted, and Brave Rewards will no longer be enabled. If you enable Brave Rewards again later, you will start with a new profile. Read our $1support article about resetting$2 for more details.',
+  resetRewardsTitle: 'Reset Brave Rewards',
```

Comment:
Should we update this string as well? I think it's the title for the Reset modal.

Reply:
I don't think we need to update `resetRewardsTitle: 'Reset Brave Rewards'`.

Reply:
I don't think we need to update the text here: 

![image](https://github.com/user-attachments/assets/4bcff0b6-8523-4cd0-8011-8d517760af6c)